File: 2024_java_basics/07-conditional-statements-advanced/tasks/07-working-hours.java
import java.util.Scanner;

public class WorkingHours {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double time = Double.parseDouble(scanner.nextLine());
        String day = scanner.nextLine();

        if (day.equals("Monday") || day.equals("Tuesday") || day.equals("Wednesday") ||
            day.equals("Thursday") || day.equals("Friday") || day.equals("Saturday")) {
            
            if (time >= 10 && time <= 18) {
                System.out.println("open");
            } else {
                System.out.println("closed");
            }

        } else {
            System.out.println("closed");
        }

        scanner.close();
    }
}

File: 2022_c#_basics/07-advanced-conditional-statements/07-working-hours.cs
using System;

namespace _07_Working_Hours
{
    internal class Program
    {
        static void Main(string[] args)
        {
            double time = double.Parse(Console.ReadLine());
            string day = Console.ReadLine();

            if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday" || day == "Saturday")
            {
                if (time >= 10 && time <= 18)
                {
                    Console.WriteLine("open");
                }

                else
                {
                    Console.WriteLine("closed");
                }
            }
            else
            {
                Console.WriteLine("closed");
            }
        }
    }
}
